src/OrchardCore.Modules/OrchardCore.Autoroute/Handlers/AutoroutePartHandler.cs

```diff
@@ -93,7 +90,11 @@ public override async Task PublishedAsync(PublishContentContext context, Autorou
 
             if (!String.IsNullOrWhiteSpace(part.Path) && !part.Disabled && part.SetHomepage)
             {
-                await SetHomeRoute(part, homeRoute => homeRoute[_options.ContentItemIdKey] = context.ContentItem.ContentItemId);
+                await SetHomeRoute(part, homeRoute =>
```

Comment:
maybe this is on the other pr?

Reply:
Yes, this is the exact same change, maybe i needed it here to test some use cases